corefxlab.sln

```diff
@@ -136,8 +136,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Text.Http.Parser", "
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Text.Http.Parser.Tests", "tests\System.Text.Http.Parser.Tests\System.Text.Http.Parser.Tests.csproj", "{793E4C5F-33B9-44AA-BE44-54EE624E79CB}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.IO.Compression.Brotli", "System.Compression.Primitives\System.IO.Compression.Brotli.csproj", "{994E5101-2E48-4818-9E32-4F68B2361198}"
```

Comment:
We are removing this project, can you get rid of 994E5101-2E48-4818-9E32-4F68B2361198 as well (i.e. https://github.com/Vedin/corefxlab/blob/testing/corefxlab.sln#L881-L892).